panoramapublic/src/org/labkey/panoramapublic/model/DataLicense.java

```diff
@@ -0,0 +1,65 @@
+package org.labkey.panoramapublic.model;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.labkey.api.data.JdbcType;
+import org.labkey.api.data.Parameter;
+import org.labkey.api.util.Link;
+
+public enum DataLicense implements Parameter.JdbcParameterValue
+{
+    CC_BY_4_0 ("CC BY 4.0", "https://creativecommons.org/licenses/by/4.0/"),
+    CC0_1_0 ("CC0 1.0", "https://creativecommons.org/publicdomain/zero/1.0/");
+
+    private final String _displayName;
+    private final String _url;
+
+    DataLicense(String displayName, String url)
+    {
+        _displayName = displayName;
+        _url = url;
+    }
+
+    public String getDisplayName()
+    {
+        return _displayName;
+    }
+
+    public String getUrl()
+    {
+        return _url;
+    }
+
+    public Link getDisplayLink()
+    {
+        return new Link.LinkBuilder(getDisplayName()).href(getUrl()).target("_blank").clearClasses().build();
+    }
+
+    public String getDisplayLinkHtml()
+    {
+        return getDisplayLink().getHtmlString().toString();
```

Comment:
Suggest making this return HtmlString to make it clearer who has been responsible for encoding.

Reply:
Done